src/Cohesity.Powershell/Scripts/Utility/Set-CohesityCmdletConfig.ps1

```diff
@@ -24,41 +25,42 @@ function Set-CohesityCmdletConfig {
         .EXAMPLE
         Set-CohesityCmdletConfig -LogSeverity 2
         Enables the log severity to 2.
+        Log severity 1 for success/failure status, 2 for data validation errors, 3 for exception messages.
         .EXAMPLE
-        Set-CohesityCmdletConfig -LogRequestedPayload $true
+        Set-CohesityCmdletConfig -LogRequestedPayload:$true
         Enables the log for request payload.
         .EXAMPLE
-        Set-CohesityCmdletConfig -LogResponseData $true
+        Set-CohesityCmdletConfig -LogResponseData:$true
         Enables the log for response data.
         .EXAMPLE
-        Set-CohesityCmdletConfig -LogHeaderDetail $true
+        Set-CohesityCmdletConfig -LogHeaderDetail:$true
         Enables the logs for headers.
         .EXAMPLE
-        Set-CohesityCmdletConfig -RefreshToken $true
+        Set-CohesityCmdletConfig -RefreshToken:$true
         Enables the flag RefreshToken, the cmdlet framework would implicitly attempt to refresh the expired token. The user does not need to explicitly connect to the cluster post token expiry.
     #>
-    [CmdletBinding(DefaultParameterSetName = 'LogSeverity', SupportsShouldProcess = $True, ConfirmImpact = "High")]
+    [CmdletBinding(DefaultParameterSetName = 'Default', SupportsShouldProcess = $True, ConfirmImpact = "High")]
     param(
-        [Parameter(Mandatory = $false, ParameterSetName = 'LogSeverity')]
+        [Parameter(Mandatory = $false)]
         [ValidateSet(0, 1, 2, 3)]
         # Set the log level.
-        $LogSeverity = $null,
-        [Parameter(Mandatory = $false, ParameterSetName = 'LogRequestedPayload')]
-        [ValidateSet($true, $false)]
-        # not recommended, the request payload may contain passwords or key information.
-        $LogRequestedPayload = $false,
-        [Parameter(Mandatory = $false, ParameterSetName = 'LogResponseData')]
-        [ValidateSet($true, $false)]
+        [int]$LogSeverity,
+        [Parameter(Mandatory = $false)]
+        # Not recommended, the request payload may contain passwords or key information.
+        [switch]$LogRequestedPayload = $false,
+        [Parameter(Mandatory = $false)]
         # Log the response data.
-        $LogResponseData = $false,
-        [Parameter(Mandatory = $false, ParameterSetName = 'LogHeaderDetail')]
-        [ValidateSet($true, $false)]
+        [switch]$LogResponseData = $false,
+        [Parameter(Mandatory = $false)]
         # Log the header details.
-        $LogHeaderDetail = $false,
-        [Parameter(Mandatory = $false, ParameterSetName = 'RefreshToken')]
-        [ValidateSet($true, $false)]
+        [switch]$LogHeaderDetail = $false,
+        [Parameter(Mandatory = $false)]
         # If set and the token has expired, the framework would attempt refreshing the token.
-        $RefreshToken = $false
+        [switch]$RefreshToken = $false,
+        [Parameter(Mandatory = $false)]
+        # Log file path. For example set the path as C:\temp
+        [string]$LogFilePath
```

Comment:
If not specified default to existing behavior.

Reply:
Its been handled in CSLog.